server.js

```diff
@@ -18,39 +24,48 @@ app.use(cors());
 app.get('/location', handleLocation);
 
 function handleLocation( request, response ) {
-  try {
-    let city = request.query.city;
-    // eventually, get this from a real live API
-    // But today, pull it from a file.
-
-    // throw 'john is ugly or something';
-
-    const url = 'https://us1.locationiq.com/v1/search.php';
-    const queryStringParams = {
-      key: process.env.LOCATION_TOKEN,
-      q: city,
-      format: 'json',
-      limit: 1,
-    };
-
-    superagent.get(url)
-      .query(queryStringParams)
-      .then( data => {
-        let locationData = data.body[0];
-        // console.log(locationData);
-        let location = new Location(city, locationData);
-        response.json(location);
-      });
-  }
-  catch(error) {
-    let errorObject = {
-      status: 500,
-      responseText: 'john is ugly or something',
-    };
-    response.status(500).json(errorObject);
-  }
+  let city = request.query.city.toLowerCase();
+  console.log('the city variable is', city);
+  // check if in sql db first
+  const SQL = `SELECT * FROM locations WHERE search_query='${city}'`;
```

Comment:
You don't want to pass your search query directly in.  You want to use safe values.
`const SQL = 'SELECT * FROM locations WHERE search_query = $1';`
`const sqlArr = [city];`
This is the proper way to go about it